x-pack/plugin/esql/compute/src/main/generated-src/org/elasticsearch/compute/aggregation/TopBooleanAggregator.java

```diff
@@ -23,6 +23,9 @@
 
 /**
  * Aggregates the top N field values for boolean.
+ * <p>
+ *     This class is generated. Do not edit it.
```

Comment:
It'd be kinder if you said `This class is generated. Edit X-TopAggregator.java to edit this file.`